pkg/apis/migration/v1alpha1/migmigration_types.go

```diff
@@ -34,6 +34,7 @@ type MigMigrationSpec struct {
 	QuiescePods     bool                  `json:"quiescePods,omitempty"`
 	KeepAnnotations bool                  `json:"keepAnnotations,omitempty"`
 	Verify          bool                  `json:"verify,omitempty"`
+	Cancel          bool                  `json:"cancel,omitempty"`
```

Comment:
The spec is declarative so the boolean should be the adjective `canceled` not the verb `cancel`.
Let's use the _American English_ spelling `canceled`.

Reply:
Done